src/app/services/login.service.ts

```diff
@@ -0,0 +1,39 @@
+import { Injectable } from '@angular/core';
+import {HttpClient} from "@angular/common/http";
+import {Observable, tap} from "rxjs";
+import {User} from "../model/user";
+import {Credentials} from "../model/credentials";
+
+@Injectable({
+  providedIn: 'root'
+})
+export class LoginService {
+
+  loginURL = 'http://localhost:3000/login';
```

Comment:
**suggestion:** you cand keep this url inside the `environment.ts` file or you could use a [proxy](https://angular.io/guide/build#proxying-to-a-backend-server)